### DISCUSSION TITLE
Plots with expression of target genes

### DISCUSSION THREAD

**Discussion by afaissa** [aertslab/SCENIC#241] [R]
Hi,
I think I'm having many transcription factors with repression activity.
Is there a way to generate plots with the expression of target genes of a given TF to show this negative association?

**Comment by afaissa** [comment]
I meant to say:
In some cases, the regulon activity does not match with expression of the transcription factor (TF) and its direct-coexpressed genes.

Does that mean the regulon activity is highest when these genes are not expressed? or is it an error in SCENIC?
If it´s not an error, is there a way to automatically select the regulons with high activity that match expression of TFs and direct-coexpressed genes? 

Any thoughts on this would be greatly appreciated, 
Thank you

**Comment by lixin4306ren** [comment]
have the same question, any suggestion?

**Comment by afaissa** [comment]
Not yet, 

I still have the same question and didn't  receive any response.

Could someone please help us with this?

**Comment by afaissa** [comment]
Not yet,

I still have the same question and didn't receive any response.

Could someone please help us with this?
